db/migrate/20210426124525_add_foreignkeys.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignkeys < ActiveRecord::Migration[6.1]
```

Comment:
are you using rails 6.1?
It was instructed to use 5.2, though dont change it now

Reply:
Sure.